tests/plotting/test_plotting.py

```diff
@@ -3811,3 +3811,104 @@ def test_voxelize_volume():
     # Create a voxel volume from unequal density dimensions and plot result.
     vox = pv.voxelize_volume(mesh, density=[0.15, 0.15, 0.5])
     vox.plot(scalars='InsideMesh', show_edges=True, cpos=cpos)
+
+
+def test_paraview_2d_style():
```

Comment:
I'm wondering if it makes sense to set `show_scalar_bar=False` for all these tests. I've been thinking lately that we out to make the image cache focus on the key feature we are testing and leave out some things like scalar bars that might be noise and overpower subtle changes that we may want to detect in the future. I don't know 🤷🏻

Reply:
I agree.  The scalar bar provides a point of reference for size and position here.  But it is more of an accident than on purpose.  It also means any subtle change to scalar bar will break these tests, which is not good.  I will check that the images are still understandable for review of behavior without them.

Reply:
Done in https://github.com/pyvista/pyvista/pull/5336/commits/b51baa89bc238caefb94c83376e135de82e8b1ab